selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/user/TestUserImpl.java

```diff
@@ -138,13 +106,6 @@ public void delete() {
             format("User name='%s' failed to remove workspace name='%s'", workspace, name), e);
       }
     }
-
-    try {
-      userServiceClient.remove(id);
```

Comment:
Is there a good reason to remove this?

Reply:
Che 6 doesn't allow to remove user by using user service so as user management is under control of Keycloak.

Reply:
Selenium tests are creating and removing test users by launcher webdriver.sh.